src/successModal/SuccessModal.style.tsx

```diff
@@ -90,7 +93,7 @@ const SuccessTitle = styled.h1`
   flex: 1;
   justify-content: center;
   margin: 0; /*reset h1*/
-  padding: ${space.xl} 0 32px; /* TheVoice spacing*/
+  padding: ${space.xl} ${space.xl} 32px; /* TheVoice spacing*/
```

Comment:
Should we use `TheVoice` instead?

Reply:
Resolved here: https://github.com/blablacar/ui-library/pull/1097/commits/f2db1bca1397f671080e9a6cbc21ada0d51a2028#diff-8414a8f16a238f7ff57de8829846aee5a2e4afda415090dd97528d8dadc09785R92